CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
+- Separate `Install-PowerShellRemoting.ps1` from `psrp.windows` nuget package. (#5330)
+- Include symbols folder an embedded zip when packaging symbols. (#5333)
+- Add Uniform Type Identifier conforming with Apple standards using a reverse DNS style prefix. (#5323)
+- Update `Wix` toolset download link to newer version 3.11 (#5339) (Thanks @bergmeister!)
+- Re-enable macOS launcher after fixing an issue that blocked macOS package generation. (#5291) (Thanks @thezim!)
+- Set expected binaries and variable name for folder for symbols build. (#5357)
+- Rename and update PowerShell `ETW` manifest to remove the Windows PowerShell dependency. (#5360)
+- Add ability to produce `tar.gz` package for Raspbian. (#5387)
+- Update `Find-Dotnet` to find dotnet with the compatible SDK. (#5341) (Thanks @rkeithhill!)
+- Add signing manifest and script to update it with production values. (#5397)
+- Add `install-powershell.ps1` to install PowerShell Core on windows. (#5383)
+- Make `-Name` a dynamic parameter in `Start-PSPackage`. (#5415)
+- Support `[package]` tag in PR CI and fix nightly build on macOS. (#5410)
+- Enhance `install-powershell.ps1` to work on Linux and macOS. (#5411)
+- Move the `RCEdit` step to the build phase rather than the packaging phase. (#5404)
+- Allow packaging from a zip package to allow for signing. (#5418)
+- Add automation to validate PowerShell Core packages using Docker containers. (#5401)
+- Try fixing the `brew update` issue in bootstrap script. (#5400)
+- Enable `install-powershell.ps1` to update the current running PowerShell Core. (#5429)
+- Add standard set of VSCode workspace setting files. (#5457) (Thanks @rkeithhill!)
+- Add support for installing PowerShell Core on Amazon Linux via `install-powershell.sh`. (#5461) (Thanks @DarwinJS!)
+- Get `PowerShellGet` and `PackageManagement` from the PowerShell Gallery. (#5452)
+- Fix `Start-PSBuild` on `WSL` if repository was already built on Windows. (#5346) (Thanks @bergmeister!)
+- Fix build in VSCode and use an improved version of `tasks.json`. (#5453)
+- Add scripts for signing packages in the release build workflow. (#5463)
+
+### Documentation and Help Content
+
+- Fix the codebase to use the consistent copyright string. (#5210)
+- Add documentation about how to create `libpsl` and `psrp.windows` nuget packages. (#5278)
+- Add help strings in PowerShell banner. (#5275) (Thanks @iSazonov!)
+- Change all links in `README.md` to absolute as they are being used in other places outside of GitHub. (#5354)
+- Update instructions to build on VSCode based on `pwsh`. (#5368)
+- Update `FAQ.md` about how to use PowerShell Core nuget packages. (#5366)
+- Correct the Fedora documentation (#5384) (Thanks @offthewoll!)
+- Add instructions about how to create the `PowerShell.Core.Instrumentation` nuget package. (#5396)
+- Updated PowerShell to use the latest help package. (#5454)
+
+### Compliance Work
+
+Some compliance works are done as required for Microsoft to continue to
```

Comment:
I don't think we need to say `some` here, just: Compliance work is required for Microsoft to continue...

Reply:
Updated as suggested.